### PR TITLE
adding "Bot:" per common practice and and welcome template

### PR DESCRIPTION
**Pull request by Steinsplitter**

### CODE REVIEW

File: qic2.py [hunk 1]
```diff
@@ -610,12 +610,12 @@ def marshalData() :
       text = page.get(get_redirect=True)
       oldtext = text
     else :
-      text = 'Welcome to commons ' + key + ". What better way than starting off with a Quality Image promotion could there be? :-) --~~~~\n\n"
+      text = '{{Welcome|realName=|name=' + key + ". What better way than starting off with a Quality Image promotion could there be? :-) --~~~~\n\n"
```
Comment (dschwen) [inline qic2.py]: Looks like you are not closing this template.

**Comment by Steinsplitter** [comment]
Wow. you are fast :)
Should be fixed in https://github.com/Steinsplitter/QICbot/commit/2d554fe51754448af5a26e36ba03186178869bd3

**Comment by dschwen** [comment]
Thanks!

**Comment by dschwen** [comment]
P.S.: I use Github for work so I'm pretty much always on (plus I have enabled notifications) :-)

**Comment by dschwen** [comment]
Ok, the changes should be live on wmflabs now. I had to merge in some commits that hadn't been on github yet. Hope it works...

**Pull request merged**